classes/Notification.php

```diff
@@ -137,6 +137,10 @@ public static function getLastElementsIdsByType($type, $idLastElement)
                 'status' => ((!empty($value['status'])) ? Tools::safeOutput($value['status']) : ''),
                 'customer_name' => $customerName,
                 'date_add' => isset($value['date_add']) ? Tools::displayDate($value['date_add']) : 0,
+                'customer_view_url' => Context::getContext()->link->getAdminLink('AdminCustomers', true, [
+                    'customerId' => $value['id_customer'],
+                    'viewcustomer' => 1,
```

Comment:
Why continue to use integer for boolean behavior?
```suggestion
                    'viewcustomer' => true,
```

Reply:
Well, first it's a legacy code and second, I don't see any reason why one would be better than the other.

If you feel other way, I can change. :)

Reply:
Just use the good type, it's a boolean so use true / false. With this kind of check `viewcustomer => '12345678978431321561a3z5d13az51d3az1da3z1d'` is also valid, but integer / string will also cost more in memory than a simple bool.
This is why it's sometimes hard to migrate MySQL to PostgreSQL because of wrong usage. If you set 1 in a pgsql boolean type, you'll get an error, whereas MySQL says "ok".

Reply:
Done, give back my PR Approval. :stuck_out_tongue: